src/Controller/ProjectController.php

```diff
@@ -76,12 +75,12 @@ public function isBillable(Request $request, Project $project, ProjectRepository
 
         if (!empty($body['value'])) {
             $project->setIsBillable($body['value']);
-            $projectRepository->save($project, true);
-
-            return new JsonResponse([$body], 200);
         } else {
-            throw new BadRequestHttpException('Value not set.');
+            $project->setIsBillable(false);
```

Comment:
I don't agree that not supplying correct data to the function should result in the project not being billable.